internal/commands/pull/inputs.go

```diff
@@ -27,7 +26,7 @@ type inputs struct {
 	IncludeDependencies bool
 	IncludeHosting      bool
 	DryRun              bool
-	TemplateID          string
+	TemplateIDs         []string
```

Comment:
[question] did Templates want to be an array input for `app create` similar to how it is here?

Reply:
User Story:

When you create a template, you choose a single template, but when you pull, you should have an option of what frontend templates you can create. I realize that this may be a bit confusing. Would calling this flag --template-client be better?

Explanation:

I think it was an oversight that this was just a string before, so I fixed it here. 

I didn't realize that this was a string until I tried to pull multiple templates without the prompt.

The prompt from before allowed users to pull multiple templates which I think we should maintain feature parity for.